include/libtorrent/kademlia/node.hpp

```diff
@@ -134,7 +134,7 @@ class TORRENT_EXTRA_EXPORT node : boost::noncopyable
 	int data_size() const { return int(m_storage.num_torrents()); }
 #endif
 
-#ifndef TORRENT_DISABLE_LOGGING
+#if TORRENT_USE_IOSTREAM
```

Comment:
with your new ``should_log`` it may be practical to actually make this print to the log. which would seem like a reason to keep it under TORRENT_DISABLE_LOGGING